DataFormats/SiPixelDigi/interface/PixelDigi.h

```diff
@@ -17,7 +17,7 @@ class PixelDigi {
   typedef unsigned int PackedDigiType;
   typedef unsigned int ChannelType;
 
-  PixelDigi( int packed_value) : theData(packed_value) {}
+  explicit PixelDigi(PackedDigiType packed_value) : theData(packed_value) {}
```

Comment:
For my education: Can you elaborate why this is needed?

Reply:
Because we pack on GPU....

Reply:
Well, this comment was not very extensive...
I am lacking experience in GPU programming and don't know the drawbacks of implicit conversion in that context.

Reply:
ah sorry. I though the construct itself.
It is simply good practice to avoid implicit conversion. Nothing to do with GPU ...

Reply:
I see, thanks!